cookbooks/bcpc-hadoop/recipes/hive_table_stat.rb

```diff
@@ -1,27 +1,29 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2 
 #
 # Cookbook Name : bcpc-hadoop
-# Recipe Name : hive_table_stat
-# Description : create database for hive table statistics 
+# Recipe Name : hive_table_stats
+# Description : Setup database  to record temporary hive table statistics
 #
 
-#mysql_connection_info = {
-#  :host     => '127.0.0.1',
-#  :username => 'root',
-#  :password => get_config('mysql-root-password')
-#}
+# Hive table stats user
+make_config('mysql-hive-table-stats-user', node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db_user"])
 
-#mysql_database node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"] do
-#  connection mysql_connection_info 
-#  action :create
-#end
-
-#mysql_database_user node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db_user"] do
-#  connection mysql_connection_info 
-#  password get_config("mysql-hive-table-stats-password") 
-#  action :create
-#end
+hive_table_stats_passwd = get_config('mysql-hive-table-stats-password')
+if hive_table_stats_passwd.nil?
+  hive_table_stats_passwd = secure_password
+end
 
+bootstrap = get_bootstrap
+hive_search = get_nodes_for("hive_config").map!{ |x| x['fqdn'] }.join(",") 
+hive_nodes = hive_search == "" ? node['fqdn'] : hive_hosts
 
+chef_vault_secret "mysql-hive-table-stats-password" do
```

Comment:
so then I also need to alter line 11: hive_table_stats_passwd = get_config('mysql-hive-table-stats-password') to be hive_table_stats_passwd = get_config('mysql-hive-table-stats')?

I am not sure I agree with the notion in the first place since we designate all items with -user and -password